evalml/utils/woodwork_utils.py

```diff
@@ -10,6 +10,8 @@
     ww.logical_types.Integer.type_string,
     ww.logical_types.Double.type_string,
     ww.logical_types.Boolean.type_string,
+    ww.logical_types.IntegerNullable.type_string,
```

Comment:
We should probably add Age/AgeNullable here? I wonder if we can get rid of this list though.